Rubberduck.Core/UI/Command/AddTestModuleCommand.cs

```diff
@@ -121,18 +121,16 @@ private string DeclarationFormatFor(string declarationFormat, string type, IUnit
 
         private IVBProject GetProject()
         {
-            using (var activeProject = _vbe.ActiveVBProject)
+            var activeProject = _vbe.ActiveVBProject;
+            if (!activeProject.IsWrappingNullReference)
```

Comment:
if it's null/wrapping null reference, you should dispose it. Just don't dispose when it's not null.

Reply:
IIUC we don't have a wrapper to dispose if it's wrapping a null reference, as such that's not necessary

Reply:
You're right. I thought we had examples of disposing after a null check but I looked and they weren't for the wrappers themselves. My bad.